### PR TITLE
fix: update event only when resizing ends on a different date

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- update event only when resizing ends on a different date

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/src/hooks/dayGridWeek/alldayGridRowEventResize.ts [hunk 1]
```diff
@@ -61,20 +61,20 @@ export function useAlldayGridRowEventResize({
...
-      draggingState === DraggingState.IDLE &&
-      !isNil(resizingEvent) &&
-      !isNil(currentGridX) &&
-      targetEventGridIndices.start <= currentGridX &&
-      targetEventGridIndices.end !== currentGridX;
-
-    if (shouldUpdateEventEnd) {
-      const targetDate = row[currentGridX];
+    const isDraggingEnd =
+      draggingState === DraggingState.IDLE && !isNil(resizingEvent) && !isNil(currentGridX);
```
Comment (jajugoguma) [inline apps/calendar/src/hooks/dayGridWeek/alldayGridRowEventResize.ts:65]: `!isNil()`을 대체하기 위해 `isPresent()`를 만들어두었습니다!

**Review by adhrinae — APPROVED** [review]

**Review by jajugoguma — COMMENTED** [review]
리뷰 완료합니다!

**Review by adhrinae — COMMENTED** [review]
기존 로직에 따르면 드래그를 좌우로 하다가 같은 위치에 놓았을 때도 드래깅 중인 이벤트를 초기화하지 않아 문제가 발생했었습니다. 이를 수정한 내용입니다.

File: apps/calendar/src/hooks/dayGridWeek/alldayGridRowEventResize.ts (hunk 1 above)
Comment (adhrinae) [reply apps/calendar/src/hooks/dayGridWeek/alldayGridRowEventResize.ts:65]: 이건 제가 반영해서 머지할게요.

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

**Pull request merged**